.github/workflows/pr.yml

```
@@ -81,19 +81,19 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         # if: github.ref == 'refs/heads/main'
         with:
-          github_token: ${{ secrets.RHDH_BOT_TOKEN }}
+          deploy_key: ${{ secrets.RHDH_BOT_TOKEN }}
           publish_branch: gh-pages
           keep_files: true
           publish_dir: ./titles-generated
 
       - name: PR comment with doc preview
         uses: actions/github-script@v7
         with:
-          github-token: ${{ secrets.RHDH_BOT_TOKEN }}
+          deploy_key: ${{ secrets.RHDH_BOT_TOKEN }}
```

```suggestion
          github-token: ${{ secrets.RHDH_BOT_TOKEN }}
```

See https://github.com/actions/github-script?tab=readme-ov-file#using-a-separate-github-token -- deploy_token is not supported for this action.